core/src/main/resources/org/jboss/jbossset/bugclerk/BZMatchStreamForBlockers.drl

```diff
@@ -0,0 +1,44 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2015, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.jbossset.bugclerk;
+
+import org.jboss.set.aphrodite.domain.Issue;
+import org.jboss.set.aphrodite.domain.Comment;
+
+import org.jboss.jbossset.bugclerk.Violation;
+import org.jboss.jbossset.bugclerk.Severity;
+
+global java.util.List list
+
+rule "BZMatchStreamForBlockers"
+  salience 0
+  dialect "mvel"
+  when
+    $parent : Candidate(!JIRA, !bug.getBlocks().isEmpty(), $children: bug.blocks ,filtered == false);
+    //this will alert on bad different status - might be better to have some method in java?
```

Comment:
I'm not sure what you mean here by "this will alert on bad different status". Do you mean it will report false+ because of the status ? Can't we in this discuss add condition on the status of the issue ?

Reply:
most likely started thought, got distracted and continued not from right pointer. Will check.